cache/spring-cloud-azure-starter/spring-cloud-azure-sample-cache/terraform/variables.tf

```diff
@@ -0,0 +1,23 @@
+variable "application_name" {
+  type        = string
+  description = "The name of your application."
+  default     = "redis-sample"
+}
+
+variable "location" {
+  type        = string
+  description = "The Azure region where all resources in this example should be created."
+  default     = "eastus"
+}
+
+variable "sample_tag_value" {
+  type        = string
+  description = "The value of spring-cloud-azure-sample tag."
+  default     = "true"
+}
+
+variable "azure_subscription_id" {
```

Comment:
I don't think it's a good approach to do this.

Reply:
https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/data-sources/subscriptions